audio/nrsc5/Portfile

```diff
@@ -32,11 +32,16 @@ depends_build-append \
                     path:bin/cmake:cmake \
                     port:libtool \
                     port:autoconf \
-                    port:automake
+                    port:automake \
+                    port:fftw-3 \
+                    port:fftw-3-single \
```

Comment:
These dependencies are confused. `depends_lib` means "needed at build time and at runtime" and so anything listed in `depends_lib` does not also need to be specified in `depends_build` and `depends_run`. Does nrsc5 link with libraries installed by fftw-3 and fftw-3-single? If so, they belong in `depends_lib`, not in the other two dependency types.